Upload model

jfan33 opened the pull request:
Machine learning models for fraud detection

jfan33 committed:
Upload model

Machine learning models for fraud detection

jfan33 committed:
Update README.md

jfan33 committed:
Update README.md

jfan33 committed:
Update README.md

jfan33 committed:
Update README.md

jfan33 committed:
Update README.md

fimuchka commented on fraudDetect-Jessie.py:
```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+import time
+
+datapath = "/Users/jessiefan/Downloads/"
```
This won't work for anyone else since the path is different. Add arguments to the file, if you need to see how look at the activity_producer.py file in the  feature/activity_producer branch

fimuchka commented:
Please rebase off of master

fimuchka reviewed (commented):
One initial comment

jfan33 committed:
updating model

jfan33 committed:
Changes

-updated readme (final model chosen)
-added model persistence in fraudDetect1-Jessie.py
-removed redundancy (fraudDetect & reuse.py)

carolinfly committed:
Add files via upload

jfan33 committed:
Update fraud detection model

Add decision tree

jfan33 committed:
Update Fraud Detection - Logistic Regression.ipynb

rename

jfan33 committed:
Edit readme.md

Update description

jfan33 committed:
add pickle

adding pkl for the new model

jfan33 committed:
Modifying models

Renaming old and new models

jfan33 committed:
Updating readme

Updating readme

jfan33 committed:
Change readme

Change readme

jfan33 committed:
Change model output

Show predictions instead of scores
